src/routes/blog/index.svelte

```diff
@@ -1,24 +1,51 @@
 <script lang="ts" context="module">
   import type { LoadInput, LoadOutput } from '@sveltejs/kit/types/internal';
 
-  export async function load({ fetch }: LoadInput): Promise<LoadOutput> {
-    const response = await fetch(`/blog.json`);
-    return { props: { posts: await response.json() } };
+  export async function load({ url }: LoadInput): Promise<LoadOutput> {
+    const target = new URL('/blog.json', url);
+    const rawLabels = url.searchParams.get('l');
+    const selectedLabels = [];
+
+    if (rawLabels?.length > 0) {
+      target.searchParams.set('l', rawLabels);
+      selectedLabels.push(...rawLabels.split(','));
+    }
+
+    const response = await fetch(target.toString());
+    const { posts, labels } = await response.json();
```

Comment:
@NLTN Looks like console.logging the result of these variables returns 
```{ posts: undefined, labels: undefined }```
And that response only contains a list of blog posts with no labels. Did you maybe forget to commit a file?

Reply:
@jaasonw I double checked, and I believe that I committed all the necessary files. BTW, everything should work smoothy no matter what a list of blog posts has labels or not.